website_signup_legal_page_required/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
```

Comment:
pls replace (c) w/ Copyright

Reply:
And please switch Antiun to Tecnativa, that is the new company